testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/suites/ManagementReadXmlTestCase.java

```diff
@@ -58,8 +62,89 @@ public class ManagementReadXmlTestCase {
     private static DomainLifecycleUtil domainPrimaryLifecycleUtil;
     private static DomainLifecycleUtil domainSecondaryLifecycleUtil;
 
+    private static void prepareExpectedServerConfiguration(String server, boolean propertyOne, int offset) throws Exception {
+        String serverConfigFileName = "expected-main-" + server + ".xml";
+        Path serverHome = new File(WildFlySecurityManager.getPropertyPrivileged("jboss.home", "toto")).toPath();
+        Path configurationDir = serverHome.resolve("standalone").resolve("configuration");
```

Comment:
We are running domain mode based tests, so better if you use the domain configuration directories instead of the standalone one.

In addition, you should find out how to do it via `domainPrimaryLifecycleUtil.getConfiguration().getJbossHome()` for example, to use the currently running instance and to avoid any side effect by using the shared target/wildfly-core server.